genome_assembly/hic-scaffolding.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Genome scaffolding with Hi-C on Galaxy Australia
+type: genome-scaffolding
+contributors: [VGP Project, Luke Silver, Anna Syme]
+description: This guide describes the steps required to scaffold your genome on the Galaxy Australia platform using HiC data
+affiliations: [Bioplatforms Australia, Galaxy Australia, Australian BioCommons, Threatened Species Initiative]
+---
+
+This How-to-Guide will describe the steps required to scaffold your genome on the Galaxy Australia platform, using HiC data, developed in consultations between the Bioplatforms Australia [Threatened Species Initiative](https://threatenedspeciesinitiative.com/), [Galaxy Australia](https://usegalaxy.org.au/), and the [Australian BioCommons](https://www.biocommons.org.au/).
+
+This workflow has been created from a Vertebrate Genomes Project (VGP) scaffolding workflow. 
+* For more information about the VGP project see the [Galaxy-VGP project page](https://galaxyproject.org/projects/vgp).
+* The VGP scaffolding workflow is hosted on [Dockstore](https://dockstore.org/workflows/github.com/iwc-workflows/Scaffolding-HiC-VGP8/main:main?tab=info). Please see that link for the workflow diagram and more information if required. 
```

Comment:
Also: https://workflowhub.eu/workflows/625

Reply:
Changed this one.